chart/templates/mayastor/csi/csi-node-daemonset.yaml

```diff
@@ -91,7 +91,7 @@ spec:
         - "--rest-endpoint=http://{{ .Release.Name }}-api-rest:8081"{{ if .Values.csi.node.restClient.enabled }}
         - "--enable-rest"{{ end }}
         - "--enable-registration"
-        - "--grpc-endpoint=$(MY_POD_IP):10199"{{ if .Values.csi.node.nvme.io_timeout }}
+        - "--grpc-endpoint=$(MY_POD_IP):{{ default .Values.csi.node.port 10199 }}"{{ if .Values.csi.node.nvme.io_timeout }}
```

Comment:
this also needs to be specified separately, I'm actually confused how this is working :/

Reply:
Right, I see this is against release/2.7 where there was no ipv6 support